examples/dash/dash-connect-pgsql/README.md

```diff
@@ -0,0 +1,19 @@
+# Dash App Connected to PostgreSQL Database
+
+Interactive Dash Application, connected to PostgreSQL database.
+
+![](app.png)
+
+To use the app, store the below information locally into your `.env` for `upload.py`. These variables can be found on the `Parameters only` section under `connection details` from your PostgreSQL dashboard.
+```
+PGHOST='YOUR_HOST'
+PGDATABASE='test'
+PGUSER='test_owner'
+PGPASSWORD='your_password'
+```
+
+run `python3 upload.py` locally to upload the datasets to your PostgreSQL, then compress and upload the below files for deployment. Make sure to specify the above environment variables in your [secrets](https://docs.cloud.ploomber.io/en/latest/user-guide/secrets.html) 
```

Comment:
Link to the `Dash` deployment guide

Reply:
[fixed](https://github.com/ploomber/doc/pull/259/commits/57121c34e1ed28974d35aa392b86415745122a53#diff-947610ca10aacde7e2a54afa9c3fbd37fea9055230c4a6222dc39decedc110afR25)